joomla/joomla-cms
# Fix link and button colors in header footer of Cassiopeia template

Rework of issue PR #41749
Pull Request for Issue #42302 

### Summary of Changes
In previous PR's the colors for the buttons and links in the header and footer where broken. This PR fixed those issues,

- Obsolete styling is removed
- Added correct colors to the header and footer buttons and links
- Fix hover state of scroll to top button

### Testing Instructions
Create 3 custom html modules in the positions: topbar / footer / bottom-b
Make sure you toggle the editor to Source code+ (Click in the editor Tools -> Source code+)
Use the code below in the modules and paste as in the Source code+ ans click Save.

```
<a href="#" class="btn btn-primary">Primary</a>
<a href="#" class="btn btn-secondary">Secondary</a>
<a href="#" class="btn btn-success">Success</a>
<a href="#" class="btn btn-danger">Danger</a>
<a href="#" class="btn btn-warning">Warning</a>
<a href="#" class="btn btn-info">Info</a>
<a href="#" class="btn btn-light">Light</a>
<a href="#" class="btn btn-dark">Dark</a>
<a href="#" class="btn btn-link">Button link</a>
<a href="#">Default Link</a>
```
Test if all button color are correct, also the hover state.
Test the frontend edit buttons
Switch the scroll to top icon to On in the template settings to test the scroll to top hover color

### Actual result BEFORE applying this Pull Request
Some buttons and links texts are not visible
![20231213-image-before](https://github.com/joomla/joomla-cms/assets/5610413/2d40f384-648b-4a73-914b-c1a81ba4b732)

### Expected result AFTER applying this Pull Request
All colors are visible
![20231213-image-after](https://github.com/joomla/joomla-cms/assets/5610413/8e627b2d-bf64-4630-8aca-4c215ff4916d)

### Link to documentations
Please select:
